ldcoolp/scripts/generate_wccfl_qualtrics_links

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+
+from os.path import dirname, exists, join
+from os import mkdir, stat
+
+from urllib.parse import quote, urlencode
+import json
+import requests
+
+import csv
```

Comment:
My commit b7fc6d9. This library is needed for solving the double quotation mark issue below.

Reply:
This is fine and as you pointed out `csv` is built-in so not extra dependency.